lib/connect-backbone-to-react.js

```diff
@@ -0,0 +1,147 @@
+const hoistStatics = require('hoist-non-react-statics');
+const { Component, PropTypes, createElement } = require('react');
+const debounceFn = require('lodash.debounce');
+const isArray = require('lodash.isarray');
+
+function getDisplayName(name) {
```

Comment:
Original goal of this was to have it be as vanilla JS as possible, with the hopes of not needing any transpiling. However as I coded more that goal was removed, and this is a relic of that original desire.

I don't really care either way. If you'd like me to change let me know!